src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
```

Comment:
The `unreadFeedbacks` array takes in the raw feedback object for the response body:

```
unreadFeedbacks = feedbacksResponse.body.filter(fb => fb.read_at === null)
```

So the attribute names are Udacity's fault 😄 . You could do the same, but I don't know what the consequences would be, so the current approach is probably best.